web/src/components/core/IssuesAlert.tsx

```diff
@@ -35,7 +38,16 @@ export default function IssuesAlert({ issues }) {
     >
       <List>
         {issues.map((i: Issue, idx: number) => (
-          <ListItem key={idx}>{i.description}</ListItem>
+          <ListItem key={idx}>
+            {i.description}{" "}
+            {i.kind === "solver" ? (
+              <Link to={PATHS.conflicts} variant="link" icon={<Icon name="build_circle" />}>
+                {_("Solve it")}
+              </Link>
+            ) : (
+              ""
+            )}
```

Comment:
NP: Instead of ternary operator, you can go for something _simpler_

`{ i.kind === "solver" && <Link to... /> }`